src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -1070,33 +1175,13 @@ const promise = makeAPICall();
 API.cancel(promise, "my error message");
 ```
 
-### Recap
-You now have an understanding of how to read your data through get and list queries. 
-
-[comment]: # (ADD TO RECAP a tie-in to real-world use and application of the topic discussed OR reiterate the benefit.)
-
+**Recap:** You now have an understanding of how to read your data through `Get` and `List` queries. 
```

Comment:
Anything to add to this recap as key take aways for real-world application?

Reply:
maybe we should to a "final pass" on all the recaps to make sure the examples align towards the end of the JTBD project. I think then the content will also better line up if we use one real-world example/setup throughout.